tests/e2e/specs/navigationWithLogin.js

```diff
@@ -0,0 +1,184 @@
+
+
+
+describe('Testing Navigation WITH Login', () => {
```

Comment:
There's honestly really no reason to test any of the "without login" functionality, cause at this point it doesn't look like there will be anything public.